rogii/build_amd64.cmake

```diff
@@ -0,0 +1,26 @@
+message(
+    STATUS
+    "ENV{ENV_INSTALL} = $ENV{ENV_INSTALL}"
+)
+
+if(NOT DEFINED ENV{ENV_INSTALL})
+    message(
+        FATAL_ERROR
+        "You have to specify an install path via `ENV_INSTALL' variable."
+    )
+endif()
+
+file(
+    TO_CMAKE_PATH
+    "$ENV{ENV_INSTALL}"
+    ROOT
+)
+
+set(
```

Comment:
We are overriding ARCH in build_common. So we don't need to declare it here. (in x86 script the same)